### ISSUE TITLE
[mpfr] build failure

### ISSUE THREAD

**Issue by acgetchell** [microsoft/vcpkg#19394]
**Host Environment**
- OS: Windows
- Compiler: Visual Studio 2019

**To Reproduce**
`./vcpkg install mpfr`

**Failure logs** 

```
Error: Building package mpfr:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: mpfr:x64-windows
  Vcpkg version: 2021-08-03-1a6d67b8150e94b7f473c6ab1c0cfdd794316a64
```

https://ci.appveyor.com/project/acgetchell/cdt-plusplus#L577

**Additional context**

This was working 3 days ago from a cached copy:

https://ci.appveyor.com/project/acgetchell/cdt-plusplus/builds/40229707#L1453

**Comment by dg0yt** [comment]
Duplicate of #19353?

**Comment by JonLiu1993** [comment]
@acgetchell ,This issue Duplicate of #19353,will be fixed by #19361
